src/component/card_count/CardCount.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./CardCount.module.scss";
+
+interface PropsCardCount {
+  count: number;
+  variant: 1 | 2;
+}
+
+const CardCount: React.FC<PropsCardCount> = ({ count, variant }) => {
+  let text = "kartki";
+
+  if (count == 0 || count >= 5) text = "kartek";
+  else if (count == 1) text = "kartka";
```

Comment:
to raczej nie zadziała przy większych liczbach ale niech sobie tak będzie xD

Reply:
Przy jakich liczbach nie zadziała?

Reply:
22 dla przykładu

Reply:
https://stackoverflow.com/a/42884713 tu częściowo jest opisane